includes/class-amp-theme-support.php

```diff
@@ -2197,6 +2197,11 @@ private static function get_optimizer_configuration( $args ) {
 			);
 		}
 
+		// Prevent styles from being removed when in Customizer preview and Standard mode is enabled.
+		if ( ! empty( $args['allow_dirty_styles'] ) && amp_is_canonical() ) {
+			$transformers = array_diff( $transformers, [ Optimizer\Transformer\AmpBoilerplate::class ] );
+		}
```

Comment:
I think this isn't the right approach. Really what should be done is fixing `AmpBoilerplate` to not remove styles that are not for the boilerplate.